packages/examples/cvat/exchange-oracle/src/core/config.py

```diff
@@ -94,32 +102,41 @@ class CvatConfig:
 
 
 class StorageConfig:
-    endpoint_url = os.environ.get("STORAGE_ENDPOINT_URL", "storage.googleapis.com")
-    region = os.environ.get("STORAGE_REGION", "")
-    access_key = os.environ.get("STORAGE_ACCESS_KEY", "")
-    secret_key = os.environ.get("STORAGE_SECRET_KEY", "")
-    data_bucket_name = os.environ.get("STORAGE_DATA_BUCKET_NAME", "")
-    results_dir_suffix = os.environ.get("STORAGE_RESULTS_DIR_SUFFIX", "-results")
-    secure = str_to_bool(os.environ.get("STORAGE_USE_SSL", "true"))
+    # common attributes
+    provider: ClassVar[str] = os.environ["STORAGE_PROVIDER"].lower()
+    data_bucket_name: ClassVar[str] = os.environ["STORAGE_RESULTS_BUCKET_NAME"]
+    endpoint_url: ClassVar[str] = os.environ[
+        "STORAGE_ENDPOINT_URL"
```

Comment:
Probably, we can use `DEFAULT_S3_HOST` and `DEFAULT_GCS_HOST` in `provider_endpoint_url` and `bucket_url`.